src/macaron/parsers/bashparser_model.py

```diff
@@ -0,0 +1,836 @@
+# Copyright (c) 2024 - 2025, Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+# Licensed under the Universal Permissive License v 1.0 as shown at https:#oss.oracle.com/licenses/upl/.
+# pylint: skip-file
+# flake8: noqa
+
+# Type definitions for Bash AST as produced (and json-serialised) by the "mvdan.cc/sh/v3/syntax" bash parser
+
+from __future__ import annotations
+
+from enum import Enum
+from typing import Any, Literal, NotRequired, Optional, TypedDict, TypeGuard, Union
```

Comment:
Looks like `Any` and `Optional` are not used, but that was not picked up because the linters were disabled.